ggshield/core/git_hooks/ci/commit_range.py

```diff
@@ -213,13 +208,13 @@ def github_actions_range(verbose: bool) -> List[str]:  # pragma: no cover
             err=True,
         )
 
-    if push_before_sha and push_before_sha != EMPTY_SHA:
-        commit_list = get_list_commit_SHA(f"{push_before_sha}...")
+    if pull_req_base_sha and pull_req_base_sha != EMPTY_SHA:
+        commit_list = get_list_commit_SHA(f"{pull_req_base_sha}..")
         if commit_list:
             return commit_list
 
-    if pull_req_base_sha and pull_req_base_sha != EMPTY_SHA:
-        commit_list = get_list_commit_SHA(f"{pull_req_base_sha}..")
+    if push_before_sha and push_before_sha != EMPTY_SHA:
+        commit_list = get_list_commit_SHA(f"{push_before_sha}...")
```

Comment:
nit: There is no change here, is there?  Just the order that changed

Reply:
The order is actually important, as in PR the push can be populated but with an wrong information. We first need to check the PR sha, then, if not populated, we're sure that we're in the case of a push.
I added a comment on this somewhere, I will add it here too